shared/middleware/channels_api.rb

```diff
@@ -283,4 +317,8 @@ class ChannelsApi < Sinatra::Base
   post %r{/v3/channels/([^/]+)/abuse/delete$} do |_id|
     call(env.merge('REQUEST_METHOD' => 'DELETE', 'PATH_INFO' => File.dirname(request.path_info)))
   end
+
+  def project_validator?
```

Comment:
Possibly worth a comment explaining this pass-through method is here so we can stub it in tests.